.github/PULL_REQUEST_TEMPLATE/pr_template.md

```
@@ -0,0 +1,17 @@
+# Description
+<!-- please describe, which changes have been made to the documentation -->
+
+## Definition of Done
+<!-- please cross out aspects below (~xyz~), that do not apply or are not relevant and justify the reason behind it -->
+- [ ] code has been self-reviewed
+- [ ] documentation outcome has been verified locally
+<!-- please remove any aspects below, that do not apply or are not relevant -->
+- [ ] major changes have been added on the homepage as new features
+- [ ] examples and screenshots have been added
+
+## Issue Reference 
```

Markdownlint (0.35.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)